src/Microsoft.DotNet.Interactive.Tests/KernelCommandNestingTests.KernelCommandResult_Events.cs

```diff
@@ -195,25 +195,25 @@ public async Task Commands_sent_within_the_code_of_another_command_publish_Stand
                                  using Microsoft.DotNet.Interactive;
                                  using Microsoft.DotNet.Interactive.Commands;
 
-                                 var result = await Kernel.Root.SendAsync(new SubmitCode("echo 123", "pwsh"));
+                                 var result = await Kernel.Root.SendAsync(new SubmitCode("echo hello-world", "pwsh"));
 
                                  result.Events
 
                                  """, "csharp"));
 
-            var returnedValueFromCSharp = result.Events
-                                                .Should()
-                                                .ContainSingle<ReturnValueProduced>()
-                                                .Which.Value;
+            // var returnedValueFromCSharp = result.Events
+            //                                     .Should()
+            //                                     .ContainSingle<ReturnValueProduced>()
+            //                                     .Which.Value;
 
-            returnedValueFromCSharp
-                .Should().BeOfType<List<KernelEvent>>()
-                .Which
+            result.Events
+                // .Should().BeOfType<List<KernelEvent>>()
+                // .Which
```

Comment:
This looks good. Thanks!

You can remove the commented out code.

Reply:
done